modules/checklist/data/endpoints/steps.php

```diff
@@ -22,6 +24,23 @@ public function get_items( $request ) {
 		return $this->get_checklist_data();
 	}
 
+	public function update_item( $id, $request ) {
+		$checklist_module = Plugin::$instance->modules_manager->get_modules( 'checklist' );
```

Comment:
I see that the `$checklist_module` appears in several places, should we define it in one place?

Reply:
Done